### DISCUSSION TITLE
nuclei scan on target list of 365 hosts stops after 13min at 6%

### DISCUSSION THREAD

**Discussion by HichamAl** [projectdiscovery/nuclei#6435] [General]
### Nuclei version:
Nuclei Engine Version: v3.2.9

## Nuclei command:
nuclei -t cves -list subdomains.txt

### Current Behavior:
nuclei starts up fine but after a few minutes it stops working and eventually just quits

### Expected Behavior:
it shouldnt quit and keep scanning the domains from the provided list

### Steps To Reproduce:
1. get a list of 365 subdomains
2. run: nuclei -t cves -list subdomains.txt
3. see it quits after a little while

**Comment by l4rm4nd** [comment]
Seeing the same behaviour when scanning multiple hosts/urls.

````
./nuclei -list targets.txt -stats -s high,critical
````

The nuclei scan just terminates at some point. 

The logs do not throw any errors. It just ends, for example like:

````
[0:04:30] | Templates: 2771 | Hosts: 1 | RPS: 107 | Matched: 1 | Errors: 55 | Requests: 3223/5979 (53%)
````

Is this maybe due to the errors detected? Or is the `-stats` just not in sync and hits 100%?

Even if I set `-mhe` to a large number or use `-nmhe`, same behaviour.

**Comment by duhang1996** [comment]
> Is this maybe due to the errors detected? Or is the `-stats` just not in sync and hits 100%?

The same situation as yours:
<img width="982" alt="Image" src="https://github.com/user-attachments/assets/cea7e177-b927-450c-a76c-36344702dc99" />

Command:
root@engine:/data/nuclei# docker run --rm -v ./:/app/ projectdiscovery/nuclei -u 172.20.3.23:8089 -t /app/nuclei-templates/ -stats

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.3.6

		projectdiscovery.io

[INF] nuclei-templates are not installed, installing...
[INF] Successfully installed nuclei-templates at /root/nuclei-templates
[WRN] Found 5 templates with runtime error (use -validate flag for further examination)
[INF] Current nuclei version: v3.3.6 (outdated)
[INF] Current nuclei-templates version: v10.1.1 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] New templates added in latest release: 154
[INF] Templates loaded for current scan: 7619
[INF] Executing 7419 signed templates from projectdiscovery/nuclei-templates
[WRN] Loading 200 unsigned templates for scan. Use with caution.
[INF] Targets loaded for current scan: 1
[INF] Running httpx on input host
[INF] Found 1 URL from httpx
[INF] Templates clustered: 1703 (Reduced 1603 Requests)
[INF] Using Interactsh Server: oast.fun

**Comment by anasbousselham** [comment]
I have the same issue when scanning multiple hosts. Nuclei ends at 25% .

**Comment by Mzack9999** [comment]
Statistics might be imprecise as for hosts deemed unresponsive the engine skips the targets and might have sync issues to update the statistics before they get printed one last time.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
